cw/capture.py

```diff
@@ -1193,21 +1263,99 @@ def capture_ecdsa_stream(ot, fw_bin, pll_frequency, capture_cfg):
     # Currently, we need to reprogram the firmware before every sign operation
     # TODO: Investigate the C code (ecc384_serial) to check if we can run multiple
     #       signing operations without reloading the firmware.
-    reset_firmware = True
+    reset_firmware = False
```

Comment:
Thanks Michael! It seems that I forgot to update these comments.

At the beginning of the project, there was an issue in the C code, and we needed to reflash the firmware for each signing operation. Now, that problem has been solved and we don't need to reflash the firmware again and again.

I just wanted to keep this reflashing option there in case we need it in the future. I updated the comments accordingly to reflect the current state.